app/web/src/newhotness/ApprovalFlow.vue

```diff
@@ -3,8 +3,8 @@
     v-if="status !== 'unexpected'"
     :class="
       clsx(
-        'flex flex-col gap-sm p-sm',
-        'rounded shadow-2xl overflow-hidden',
+        'flex flex-col gap-sm p-sm overflow-y-auto',
```

Comment:
In the worst case that the button can't be viewed, allow the overflow. This is an escape hatch in case this happens again and will not appear unless a similar bug happens.